scripts/init.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -euo pipefail
+IFS=$'\n\t'
+
+packageName=${1:-}
+
+ROOT_PACKAGE_NAME=$(jq -r ".name" < package.json)
```

Comment:
You don't need any redirection, the following will work just fine: `jq --raw-output '.name' package.json`

Reply:
do we need to install `jq` locally? 😼

Reply:
`./scripts/init.sh: line 9: jq: command not found` ='(

Reply:
@shuse2 yeap. Will told me that jq is already a dependency for lisk core. So `brew install jq` .